post.html

```diff
@@ -123,34 +140,50 @@ <h4 class="post__title">
                 Image caption centered this way and I’ll make this a bit longer to indicate the amount of line-height.
             </figcaption>
         </figure>
-        <p class="post__body">
+        <p class="post__body post__body_last">
```

Comment:
you have `:last-of-type` and `:last-child` pseudo classes, you can use them instead of adding new class to html